src/main/starlark/builtins_bzl/common/objc/j2objc_library.bzl

```diff
@@ -162,7 +161,6 @@ The list of additional JRE emulation libraries required by all Java code transla
 <code>j2objc_library</code> rule. Only core JRE functionality is linked by default.""",
         ),
     } | common_attrs.CC_TOOLCHAIN_RULE,
-    cfg = apple_crosstool_transition,
```

Comment:
not sure about this one and how it affects google since AFAIUI there are no public users of j2objc

Reply:
We also need this internally regardless of whether we remove the objc_library transition.

Reply:
is there any way to reconcile that with internal vs external? this is the only reason bazel ships with a dependency on apple_support

Reply:
Long term, j2objc_library.bzl is being migrated to a newer, internal version, and this version will be deleted.

Perhaps when we are ready to remove objc_library transition internally, we can look into getting rid of this transition also.